doc/nrf/images/zigbee_signal_handler_08_deep_sleep.uml

```diff
@@ -0,0 +1,14 @@
+@startuml
+
+:ZB_COMMON_SIGNAL_CAN_SLEEP;
+partition zb_sleep_now() {
+partition zb_nrf52840_sleep() #AntiqueWhite {
+        :zb_nrf52840_priph_disable();
```

Comment:
To be improved:
 - All functions starting from `zb_nrf52840_` prefix were renamed in NCS.
 - Both `periph_enable` and `periph_disable` removed from NCS: ZBOSS no longer manages power state of shared resources. It disables `RADIO` and `TIMER/COUNTER`, but not from the `sleep` function, but from the core code.
 - `zb_osif_wait_for_event()` replaced by `zigbee_event_poll(tmo)`